catroid/src/org/catrobat/catroid/ui/ScriptActivity.java

```diff
@@ -469,6 +470,11 @@ public boolean onKeyDown(int keyCode, KeyEvent event) {
 			if (scriptFragment.getAdapter().isBackPackActionMode()) {
 				scriptFragment.getAdapter().setIsBackPackActionMode(false);
 			}
+			AddBrickFragment addBrickFragment = (AddBrickFragment) getFragmentManager().findFragmentByTag(AddBrickFragment.ADD_BRICK_FRAGMENT_TAG);
+			if (addBrickFragment != null && !addBrickFragment.isVisible() || addBrickFragment == null) {
```

Comment:
Am I wrong or can this be simplified to `if (addBrickFragment == null || !addBrickFragment.isVisible())`?